client/src/_playground/Buttons.playground.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, ButtonVariant } from '@components/Button/Button';
+
+export const ButtonsPlayground = () => (
```

Comment:
Why arrow function instead of regular one?

Reply:
Just a playground - I didn't pay much attention to it. I can change it :)